src/components/molecule/BarChartView.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { VictoryContainer, VictoryBar } from 'victory'
+
+interface Idata {
```

Comment:
- Is all data optional? I assume the chart expects some of the data to always be there.
- Is this data structure's going to be used in other widgets in the near future?